recipe/meta.yaml

```diff
@@ -42,6 +45,10 @@ requirements:
     - scrypt
     - license-expression
     - libarchive
+  run_constrained:
+    # For more details about `shellcheck`, please see this issue.
+    # xref: https://github.com/conda-forge/conda-smithy-feedstock/issues/248
+    - shellcheck
```

Comment:
`run_constrained` without a version here is a no-op right?

Reply:
Yeah it is more documentation than anything

Reply:
There was some support for doing this above ( https://github.com/conda-forge/conda-smithy-feedstock/pull/246#discussion_r1115174668 ). So had made this change

No strong feelings about other such changes

Reply:
Do we want to make a change here before merging or are we happy to merge?

Reply:
I'm fine either way

Reply:
Cool, let's merge then. If we decide to handle this another way, we can do that in a new PR